command-snapshot.json

```diff
@@ -1 +1,17 @@
-[]
+[
+  {
+    "command": "deploy:pipeline",
+    "plugin": "@salesforce/plugin-devops-center",
+    "flags": [
+      "branch-name",
+      "bundle-version-name",
```

Comment:
I could not find this command in any of the WI but I think it is OK as it is in the docs.